### ISSUE TITLE
Published JavaDoc for `kiama_3` is empty

### ISSUE THREAD

**Issue by michiexile** [inkytonik/kiama#37]
Both in the Maven-distributed JavaDoc JAR and on Javadoc.io is the actual scaladoc for the package `kiama_3` completely empty, with no documentation included.

**Comment by inkytonik** [comment]
Yes, that's true. At the time I was doing the Scala 3 distribution I lacked the time to work out why that empty doc JAR was being generated. I'll see if I can find some time to look into it again.

In the meantime, it's worth noting that there are no essential differences between the Scala 3 version of Kiama and the Scala 2 version, so the docs for the latter are a workaround for this issue.
